draft-ietf-ppm-dap.md

```diff
@@ -767,30 +767,35 @@ The order of the encrypted input shares appear MUST match the order of the
 task's `aggregator_endpoints`. That is, the first share should be the leader's,
 the second share should be for the first helper, and so on.
 
-The leader responds to well-formed requests to `[leader]/upload` with HTTP status
-code 200 OK and an empty body. Malformed requests are handled as described in {{errors}}.
+The leader responds to well-formed requests to `[leader]/upload` with HTTP
+status code 200 OK. Malformed requests are handled as described in {{errors}}.
 Clients SHOULD NOT upload the same measurement value in more than one report if
-the leader responds with HTTP status code 200 OK and an empty body.
+the leader responds with HTTP status code 200 OK.
 
 The leader responds to requests whose leader encrypted input share uses an
 out-of-date `HpkeConfig.id` value, indicated by `HpkeCiphertext.config_id`, with
-HTTP status code 400 Bad Request and an error of type 'outdatedConfig'. Clients SHOULD invalidate any
-cached aggregator `HpkeConfig` and retry with a freshly generated Report. If
-this retried report does not succeed, clients MUST abort and discontinue
-retrying.
+HTTP status code 400 Bad Request and an error of type 'outdatedConfig'. Clients
+SHOULD invalidate any cached aggregator `HpkeConfig` and retry with a freshly
+generated Report. If this retried report does not succeed, clients SHOULD abort
+and discontinue retrying.
 
-The Leader MUST ignore any report pertaining to a batch that has already been
+If the leader determines that a report is a replay based on the conditions in
+{{anti-replay}}, the report MUST be ignored and the leaader SHOULD alert the
+client with error "reportRejected".
```

Comment:
Reviewing Chris' #382 reminded me to revisit this change. David's point about a circular reference is a good one. What I want to do is repurpose {{anti-replay}} to discuss the criteria for determining if a report is a replay, but then leave it up to either {{upload-flow}} or {{early-input-share-validation}} to specify what should be done when a replay is detected. So I'll do that after #382 lands, since that PR is changing the text of {{anti-replay}}.